### DISCUSSION TITLE
"Best Practice" (อาจจะ)ไม่มีจริง | First Sutham

### DISCUSSION THREAD

**Discussion by giscus** [heyfirst/heyfirst.co#7] [General]
# "Best Practice" (อาจจะ)ไม่มีจริง | heyfirst.co

ในโลกของ Software/Technology มักจะมีบทความหรือการบอกเล่าเกี่ยวกับ best practice จุดที่น่าสนใจคือ คนที่เขียนบทความเหล่านั้นหรือคนที่พูด best practice เหล่านั้น เค้าเป็นใคร? มาจากไหน? ทำงานแบบไหน? ทำไมมาถึงบอกว่า มันดีที่สุด

http://localhost:3000/blog/best-practice-might-be-wrong

**Comment by heyfirst** [comment]
Yay! เจ๋งงะ
